### PR TITLE
feat: stargate: Change did sign marshaller json to protobuf

### PR DESCRIPTION
**Pull request by gyuguen**
Change did sign marshaler json to protobuf.
Because we fade out to amino marshaler.

### CODE REVIEW

File: x/did/keeper/msg_server_did.go [hunk 1]
```diff
@@ -65,8 +65,11 @@ func (m msgServer) DeactivateDID(goCtx context.Context, msg *types.MsgDeactivate
 		return nil, sdkerrors.Wrapf(types.ErrDIDDeactivated, "DID: %s", msg.DID)
 	}
 
-	signableDID := types.SignableDID(msg.DID)
-	newSeq, err := VerifyDIDOwnership(signableDID, docWithSeq.Seq, docWithSeq.Document, msg.VerificationMethodID, msg.Signature)
+	doc := types.DIDDocument{
```
Comment (gyuguen) [inline x/did/keeper/msg_server_did.go:68]: Instance of using the did, use the DIDDocument

Comment (gyuguen) [reply x/did/keeper/msg_server_did.go:68]: Because did is string. But not implements protobuf marshal

Comment (youngjoon-lee) [reply x/did/keeper/msg_server_did.go:68]: Yeah. It makes sense even though the user must get a DID document before deactivating it.

**Review by youngjoon-lee — APPROVED** [review]
LGTM

File: x/did/keeper/msg_server_did.go (hunk 1 above)
Comment (youngjoon-lee) [reply x/did/keeper/msg_server_did.go:68]: Yeah. It makes sense.

**Pull request merged**